contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -4,18 +4,45 @@ pragma solidity >=0.8.7 <0.9.0;
 import '../../interfaces/IDCAStrategies.sol';
 
 abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHandler {
+  mapping(uint80 => Strategy) internal _strategies;
```

Comment:
Instead of using this struct, I think we can do something like:
**1. Create a new struct**
```
struct StrategyOwnerAndVersion {
    address owner;
    uint96 latestVersion;
}
```
Why am I suggesting this? First, Solidity handles storage in slots of 32 bytes. Addresses are 20 bytes long, so we have 12 bytes (or 96 bits) for the version. Also, we will almost never try to read the name of a Strategy on-chain. It will be used mostly for off-chain purposes, so it makes sense to split it
_Note: this struct should be internal and it should not be on the interface, since it's only used a way to store data_ 

**2. Create a new mapping**
We can create a new mapping, `mapping(uint80 => string)` for the names